client/src/controllers/SkipLinkController.ts

```diff
@@ -0,0 +1,35 @@
+import { Controller } from '@hotwired/stimulus';
+
+/**
+ * Appears at the top left corner of the admin page with the tab button is clicked.
+ * Used to provide an accessible skip button for keyboard control so that users can
+ * easily navigate to the main content without having to navigate a long list of navigation links.
+ *
+ * Inspired by https://github.com/selfthinker/dokuwiki_template_writr/blob/master/js/skip-link-focus-fix.js
+ *
+ */
+
+export class SkipLinkController extends Controller<HTMLElement> {
+  skipToTarget!: HTMLElement | null;
+
+  connect() {
+    this.skipToTarget = document.querySelector(
+      this.element.getAttribute('href') || 'main',
+    );
+  }
+
+  handleBlur() {
+    if (!this.skipToTarget) return;
+    this.skipToTarget.removeAttribute('tabindex');
+    this.skipToTarget.removeEventListener('blur', this.handleBlur);
+    this.skipToTarget.removeEventListener('focusout', this.handleBlur);
+  }
+
+  skipToMain() {
```

Comment:
let's just call this `skip` instead of `skiptToMain` - the to main part adds length and it also is not necessarily true based on what values are passed into the controller.

```suggestion
  skip() {
```